src/validation.cpp

```diff
@@ -2120,6 +2120,13 @@ bool CChainState::FlushStateToDisk(
             nLastFlush = nNow;
             full_flush_completed = true;
         }
+        TRACE6(utxocache, flush,
+               (int64_t)(GetTimeMicros() - nNow.count()), // in microseconds (µs)
```

Comment:
what's the purpose of this argument?

Reply:
The purpose was to get the duration of the flush, this value can be used for benchmarking purposes. 

The `nNow` was declared here:
https://github.com/bitcoin/bitcoin/blob/384d07601185d0779c324392e798da234163ca72/src/validation.cpp#L1969